src/seedu/addressbook/parser/Parser.java

```diff
@@ -74,34 +75,36 @@ public Command parseCommand(String userInput) {
         final String arguments = matcher.group("arguments");
 
         switch (commandWord) {
+            case SortCommand.COMMAND_WORD:
+                return new SortCommand();
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
```

Comment:
You shouldn't change the indentation as it doesn't follow our coding standard now..